data/datasource_aws_asm.go

```diff
@@ -0,0 +1,55 @@
+package data
+
+import (
+	"path"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/pkg/errors"
+
+	gaws "github.com/hairyhenderson/gomplate/aws"
+)
+
+// awsSecretsManagerGetter - A subset of SSM API for use in unit testing
```

Comment:
copy-paste error, I think

```suggestion
// awsSecretsManagerGetter - A subset of Secrets Manager API for use in unit testing
```